mlflow/tracking/fluent.py

```diff
@@ -1342,32 +1332,12 @@ def _get_experiment_id_from_env():
 
 
 def _get_experiment_id():
-    # TODO: Replace with None for 1.0, leaving for 0.9.1 release backcompat with existing servers
-    deprecated_default_exp_id = "0"
```

Comment:
We have a TODO here to replace this with None. I think we should address it here instead of kicking the can down the road.

Reply:
https://github.com/mlflow/mlflow/pull/5309#discussion_r793101898
Corey suggested that this would be a backwards incompat change and we would need to do this in 2.0

Reply:
SGTM. I guess we missed the 1.0 train.